NORTRIP_read_commandline.f90

```diff
@@ -56,25 +56,25 @@ subroutine read_NORTRIP_commandline
 
     !Set saving start date string
     character_length = LEN_TRIM(start_date_input_str)
-    if (character_length >= 10) then
+    if (character_length >= 16) then
```

Comment:
Now requires minute input, i.e. runscripts needs to be updated if they don’t already.